package.json

```diff
@@ -58,7 +58,7 @@
     "grunt-jsxgettext": "1.1.0",
     "grunt-open": "0.2.3",
     "grunt-shell": "2.1.0",
-    "grunt-zanata-js": "1.3.2",
+    "grunt-zanata-js": "^1.4.1",
```

Comment:
Hi @tagoh the update of the dep needs to be done via the npm manager as the following example.

````
E.g ( npm install --save --save-exact browserify@12.0.0 )
````

Since it is dev dep. 

````
E.g ( npm install --save-dev --save-exact browserify@12.0.0 )
```

Reasons:
* The version needs to be fixed and not with ^
* The lock file needs to be updated too and we should not do it manually.